### DISCUSSION TITLE
What should be our tech stack ❓

### DISCUSSION THREAD

**Discussion by sammaji** [ExhibitHQ/ExhibitHQ-old#1] [Q&A] [answered]
Big Question, what should be our tech stack?

1. Frontend Framework
2. Styling Library
3. Any UI library?

For ease of use, we'll be using JSON  for storing data.
@Arindam200 @Swpn0neel @priyankarpal - drop your suggestions.

**Comment by Swpn0neel** [comment]
Svelte + Vanilla CSS ??

**Reply by Arindam200** [reply]
React/Next Js + Tailwind?

**Reply by sammaji** [reply]
My preferred is NextJS + Turbo and Tailwind. But we can do react or svelte.

**Reply by priyankarpal** [reply]
NextJs+ Tailwind

**Comment by sammaji** [comment, marked as answer]
So its decided, we'll use next and tailwind. I'll create a template quickly.
